README.md

```diff
@@ -103,6 +103,55 @@ always flatly available on the disk. diskv will never do anything that would
 prevent you from accessing, copying, backing up, or otherwise interacting with
 your data via common UNIX commandline tools.
 
+## Advanced path transformation
+
+If you need more control over the file name written to disk or if you want to support
+slashes in your key name or special characters in the keys, you can use the
+AdvancedTransform property. You must supply a function that returns
+a special PathKey structure, which is a breakdown of a path and a file name. Strings
+returned must be clean of any slashes or special characters:
+
+```go
+func AdvancedTransformExample (key string) *diskv.PathKey {
```

Comment:
Can you run gofmt on these examples? I'm seeing e.g. an extraneous space between the func name and the opening parens.

Reply:
That's GitHub formatting I am afraid. If you look at the raw file you'll see it too. It happens in your examples too.